src/client/Microsoft.Identity.Client/Cache/Prototype/IdentityCacheWrapper.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Identity.ServiceEssentials;
+using Microsoft.Identity.ServiceEssentials.IdentityCache;
+using Microsoft.IdentityModel.Abstractions;
```

Comment:
Is the dependency on Wilson for IIdentityLogger?

Reply:
Yes, and ITelemetryClient.